### PR TITLE
Parallel coordinates plot

### PR DESCRIPTION
**Pull request by ritikkumar1212**
## Description

This PR introduces a parallel coordinates plot visualization using Plotly.js to help visualize experiment hyperparameters and performance metrics in Visdom.

The visualization allows users to explore relationships between multiple experiment parameters such as:
- learning_rate
- batch_size
- accuracy
- loss

Each experiment is represented as a line across multiple axes, enabling easy comparison and pattern identification.

---

## Motivation and Context

Currently, Visdom lacks a way to visually analyze relationships between experiment parameters and outcomes.

This feature enables:
- Better understanding of hyperparameter impact
- Visual comparison of multiple experiments
- Improved experiment analysis workflow

This builds upon previous work on experiment tagging and query-based filtering, extending Visdom toward a complete experiment management system.

---

## How Has This Been Tested?

- Ran Visdom server locally using:
  ```bash
  PYTHONPATH=py python3 -m visdom.server.run_server

## Screenshots (if appropriate):
<img width="1901" height="819" alt="image" src="https://github.com/user-attachments/assets/e3bd9dae-9cd2-4c33-9a20-824fe236b2e3" />


## Types of changes
<!--- What types of changes does your code introduce? Put an `x` in all the boxes that apply: -->
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] Code refactor or cleanup (changes to existing code for improved readability or performance)

## Checklist:
<!--- Go over all the following points, and put an `x` in all the boxes that apply. -->
<!--- If you're unsure about any of these, don't hesitate to ask. We're here to help! -->
- [ ] I adapted the version number under `py/visdom/VERSION` according to [Semantic Versioning](https://semver.org/)
- [x] My code follows the code style of this project.
- [x] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.

### CODE REVIEW

**Comment by ritikkumar1212** [comment]
I’ve opened an issue to discuss this feature and align with the project direction before proceeding further.
